spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java

```diff
@@ -46,7 +44,7 @@
     private Repository repository;
 
     public enum Repository {
-        REDIS, CONSUL
+        REDIS, CONSUL, SPRING_DATA
```

Comment:
Even thou this implementation will be using `spring-data`, this naming here is more about the storage itself and not which framework will be used.
Maybe it can be `SQL` or `JDBC` I don't know.